### PR TITLE
SAK-34211 Questions tab and calculated questions: It shows wrong answers despite the answers are right

### PR DESCRIPTION
**Pull request by maramar7**

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java [hunk 1]
```diff
@@ -548,22 +548,39 @@ public boolean questionScores(String publishedId, QuestionScoresBean bean,
 				results.setItemGradingArrayList(answerList);
 				// The list is sorted by item id so that it will come back from the student in a 
 				// predictable order. This is also required by the getCalcQResult method.
-				Collections.sort(answerList, new Comparator<ItemGradingData>() {
-					public int compare(ItemGradingData i1, ItemGradingData i2) {
+				if (bean.getTypeId().equals("15")) { // CALCULATED_QUESTION
```
Comment (jonespm) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java]: This should import and use
import org.sakaiproject.tool.assessment.data.ifc.shared.TypeIfc;

TypeIfc.CALCULATED_QUESTION

Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java]: And it should be on the right side of the comparison as it's a constant!

Comment (maramar7) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java]: Done

Comment (raulsv) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java]: Hi @ern @mpellicer @juanjmerono @jonespm 

The requested changes are done by @maramar7. Could you please merge this PR? Thanks!

**Comment by bjones86** [comment]
@jonespm / @ottenhoff  / @ern, requested changes have been made. LGTM, care to re-review?

**Pull request merged**